src/App.tsx

```diff
@@ -1,76 +1,121 @@
 import { MantineProvider } from "@mantine/core";
+import "@mantine/core/styles.css";
+import AOS from "aos";
+import "aos/dist/aos.css";
+import { useEffect, useState } from "react";
 import {
   RouteObject,
   RouterProvider,
   createBrowserRouter,
 } from "react-router-dom";
 import Content from "./components/Content";
-import Error404 from "./pages/404/Error404";
-import Document from "./pages/Documents/Document";
-import Download from "./pages/Download/Download";
-import Overview from "./pages/Overview/Overview";
-import Pricing from "./pages/Pricing/Pricing";
+import { getDocument, getDocumentID } from "./database";
+import { LegalDocument } from "./database/models";
+import Document from "./pages/Docs/Document";
+import Docs from "./pages/Docs/MainMenu";
+import { Error404 } from "./pages/Errors/404";
+import Home from "./pages/Home";
+import Pricing from "./pages/Pricing";
 
-const routes: RouteObject[] = [
-  {
-    path: "/",
-    element: <Content />,
-    children: [
-      {
-        path: "/",
-        element: <Overview />,
-      },
-      {
-        path: "/pricing",
-        element: <Pricing />,
-      },
-      {
-        path: "/download",
-        element: <Download />,
-      },
-      {
-        path: "/privacy",
-        element: <Document id="cc62zlogyvcywte" />,
-      },
-      {
-        path: "/tos",
-        element: <Document id="u8rco7fc6upbulf" />,
-      },
-      {
-        path: "*",
-        element: <Error404 />,
-      },
-    ],
-  },
-];
+export default function App() {
+  const [loadedDocuments, setLoadedDocuments] = useState<LegalDocument[]>([]);
+
+  const routes: RouteObject[] = [
```

Comment:
Also why dont we bring this outside the component?

Reply:
i think so

Reply:
> Also why dont we bring this outside the component?

react hooks can only be used inside a component function

Reply:
> > Also why dont we bring this outside the component?
> 
> react hooks can only be used inside a component function

I think he mean the routes variable

Reply:
> > > Also why dont we bring this outside the component?
> > 
> > 
> > react hooks can only be used inside a component function
> 
> I think he mean the routes variable

there is a hook call inside that routes list

Reply:
> > > > Also why dont we bring this outside the component?
> > > 
> > > 
> > > react hooks can only be used inside a component function
> > 
> > 
> > I think he mean the routes variable
> 
> there is a hook call inside that routes list

hmm I think that we can make this resolve but just wait for @tobycm's reply

Reply:
> > Also why dont we bring this outside the component?
> 
> react hooks can only be used inside a component function

ah ok i see